src/components/Common/Button.js

```diff
@@ -1,29 +1,13 @@
 import cx from "classnames";
 import "./Button.css";
 
-export default function Button({ href, imgSrc, children, onClick, className, size = "lg", align = "center", ...rest }) {
+export default function Button({ href, imgSrc, children, className, size = "lg", align = "center", ...rest }) {
```

Comment:
mm, why do we turn a button into a link, without the onClick Prop?

Reply:
Updated the component to `ButtonLink` and also converted it to `tsx`. Please have a look and let me know your thougths.

Reply:
> mm, why do we turn a button into a link, without the onClick Prop?

so far Button component was used for "link-like" buttons only. e.g. on /earn page it's just `<button className="App-button-option App-card-option" onClick={} />`

i think it's better to have one universal Button component that accepts either url or onClick. and it can be reused everywhere. so maybe keep name "Button" and in some future we will extend its functionality and use it in components like StakeV2

makes sense?

Reply:
Yeah it does. Updated Button component to support both as of now will change or update it in future based on our need.